x-pack/plugins/security_solution/scripts/endpoint/agent_emulator/services/action_responder.ts

```diff
@@ -5,7 +5,7 @@
  * 2.0.
  */
 
-import { set } from 'lodash';
+import { set } from '@kbn/safer-lodash-set';
```

Comment:
Q: I notice that in the PR `set` is imported sometimes from `@kbn/safer-loadsh-set` and in other instances, it is imported from `@kbn/safer-loadsh-set/fp`. Is there a reason for this? 🤔 

If it is supposed to be imported from `@kbn/safer-loadsh-set/fp` then perhaps the eslint message should be updated accordingly.

Reply:
The `@kbn/safer-lodash-set/fp` module is functional equivalent to the `lodash/fp` module. So I've just replaced all the locations where we import from `lodash/fp` with `@kbn/safer-lodash-set/fp`. I guess we could update the error message to reflect that (the [`README.md`](https://github.com/elastic/kibana/blob/main/packages/kbn-safer-lodash-set/README.md#api) for `@kbn/safer-lodash-set` does also mention this).

I'm pushing a few new commits soon to fix the failing tests, I'll incorporate your suggestion there 👍

Reply:
Fixed 👍